.github/workflows/react-components.yml

```diff
@@ -8,6 +8,7 @@ on:
       - main
 env:
   CI: true
+  NODE_OPTIONS: "--max-old-space-size=8192"
```

Comment:
Can you explain why this is needed? It can mean there are memory leaks if we need to resort to using this.

Reply:
Oh sorry, the comment went away with the rebase. Sure, yeah it can mean that, but in this case, I think it's just a node 12 "feature" the default it's 512mb. From node 12 > it configures the heap limits based on more sensible defaults. Regarding the 8192, I just set the maximum number for not having problems in the future. Or we can reduce the number and then increase due to need.

I'll try to run it without this option now (I may consume fewer resources now that we fix the material-ui bundle)

Reply:
It's working without the flag now.